sync/class.jetpack-sync-actions.php

```diff
@@ -293,6 +294,20 @@ static function add_woocommerce_sync_module( $sync_modules ) {
 		return $sync_modules;
 	}
 
+	static function initialize_wp_super_cache() {
+		//WPCACHEHOME looks like a reasonable constant to test for the plugin's presence as per https://github.com/Automattic/wp-super-cache/blob/master/wp-cache.php
+		if ( false === defined( 'WPCACHEHOME' ) ) {
```

Comment:
WPCACHEHOME should be good to use, but if you want to double check, use $wp_super_cache_debug too - it should be set to 0 or 1 and is in the sample config file so it's defined even if the user hasn't configured the plugin yet.